### PR TITLE
NOTICK fix smoke tests

### PR DESCRIPTION
**Pull request by blsemo**
Add schema set-up process for smoke tests running full-fat nodes.

### CODE REVIEW

**Comment by blsemo** [comment]
Run tests: smoke integration-slow

**Comment by blsemo** [comment]
Run tests: smoke slow

**Comment by blsemo** [comment]
Run tests: smoke, slow

**Comment by blsemo** [comment]
Run tests: smoke

**Review by rnicoll — DISMISSED** [review]
Makes sense. In an ideal world we'd have pre-generated database files we used, but this not being an ideal world, I'll take fixed code over ideal.

**Review by rekalov — DISMISSED** [review]

**Comment by blsemo** [comment]
Run tests: smoke

File: testing/smoke-test-utils/src/main/kotlin/net/corda/smoketesting/NodeProcess.kt [hunk 1]
```diff
@@ -33,6 +35,7 @@ class NodeProcess(
     companion object {
         const val CORDAPPS_DIR_NAME = "cordapps"
         private val log = contextLogger()
+        private val schemaCreationTimeOutSeconds: Long = 180
```
Comment (rnicoll) [inline testing/smoke-test-utils/src/main/kotlin/net/corda/smoketesting/NodeProcess.kt]:
```suggestion
        private const val schemaCreationTimeOutSeconds: Long = 180
```
Total nitpick I know

**Comment by chrisr3** [comment]
Just wondering, but does this "schema migration" change break DemoBench too?

**Comment by blsemo** [comment]
> 
> 
> Just wondering, but does this "schema migration" change break DemoBench too?

Not impossible - I'll add it to my list of things to be checked before committing to the release branch.

**Review by rnicoll — DISMISSED** [review]

**Comment by blsemo** [comment]
Run-tests: slow, smoke

**Review by rnicoll — APPROVED** [review]

**Comment by blsemo** [comment]
Run tests: slow, smoke

**Pull request merged**